plugins/PowerManagement/PowerManagementLinux.py

```diff
@@ -0,0 +1,70 @@
+'''
+Created on 10.01.2017
+
+@author: thopiekar
+'''
+
+import dbus
```

Comment:
I would personally prefer to use Qt's DBus module instead, so we only need one DBus implementation.